src/gui/MainWindow.cc

```diff
@@ -91,13 +91,11 @@
 #include <QTemporaryFile>
 #include <QDockWidget>
 #include <QClipboard>
-#include <QDesktopWidget>
```

Comment:
This is needed for Windows builds.

Reply:
Looks promising, according to the docs `QScreen` is available since 5.0, so that should work for both versions too.